CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Note: This CHANGELOG is only for the monitoring team to track all monitoring related changes. Please see OpenShift release notes for official changes.
 
+## 4.16
+
+- [#2046](https://github.com/openshift/cluster-monitoring-operator/pull/2046) Drop TechPreview gate for CollectionProfiles.
```

Comment:
IIUC we added this as a TechPreviewNoUpgrade FeatureGate so for this PR  there is no user change log?

Reply:
ACK, the TP was replaced by the newly introduced ways of using feature-gates in https://github.com/openshift/cluster-monitoring-operator/pull/2153. At the moment there is no changelog entry for this, but we might want to point out to users that this is available in TP behind a certain FG if they want to use it. WDYT?